### ISSUE TITLE
Model zoo Pascal Voc Preserving aspect ratio

### ISSUE THREAD

**Issue by AllentDan** [qubvel/segmentation_models.pytorch#233]
Hi, qubvel. Thank you for the great job you have done, it helped me a lot. But I am a little confused that in your model zoo for Pascal VOC segmentation, you preserved aspect ratio of test image. However, for deeplabv3+, it gives me error that for "torch.cat([aspp_features, high_res_features], dim=1)". So, have you croped the image or implemented other operation after resizing while keeping aspect ratio?

**Comment by qubvel** [comment]
Hi, I have used zero padding, and ignore zero values in metric calculation.

**Comment by AllentDan** [comment]
Hi, @qubvel . It is interesting that in your repo, the U-Net architecture outperforms deeplabv3+ for Pascal Voc segmentation. Hope these questions won't bother you. So, you made experiments under the same setting? Or your U-Net applied attention?  Why are the decoder channels set to [256, 256, 256, 21, 21], rather than the default?
